core/src/test/scala/org/broadinstitute/dsde/rawls/monitor/WorkspaceMigrationMonitorSpec.scala

```diff
@@ -0,0 +1,413 @@
+        )
+      }
+
+      // Creating the temp bucket requires that the new google project has been created
+      val attempt = runAndWait(
+        WorkspaceMigrationMonitor.workspaceMigrations
+          .filter(_.workspaceId === v1WorkspaceCopy.workspaceIdAsUUID).result
+      )
+        .head
+        .copy(newGoogleProjectId = GoogleProjectId(destProject).some)
+
+      val writeAction = GoogleStorageService.resource[IO](pathToCredentialJson, None, Option(serviceProject)).use { googleStorageService =>
+        for {
+          res <- WorkspaceMigrationMonitor.createTempBucket(attempt, v1WorkspaceCopy, googleStorageService)
+          (bucketName, writeAction) = res
+          loadedBucket <- googleStorageService.getBucket(GoogleProject(destProject), bucketName)
+          _ <- googleStorageService.deleteBucket(GoogleProject(destProject), bucketName).compile.drain
+        } yield {
+          loadedBucket shouldBe defined
+          writeAction
+        }
+      }.unsafeRunSync
+
+      runAndWait(writeAction) shouldBe()
+    }
+  }
+
+  "deleteWorkspaceBucket" should "delete the workspace bucket and record when it was deleted" in {
+    val sourceProject = "general-dev-billing-account"
+    val destBucket = "v1-migration-test-" + UUID.randomUUID.toString.replace("-", "")
+
+    val v1Workspace = minimalTestData.v1Workspace.copy(
+      namespace = sourceProject,
+      googleProjectId = GoogleProjectId(sourceProject),
+      bucketName = destBucket
+    )
+
+    withMinimalTestDatabase { _ =>
+      runAndWait {
+        DBIO.seq(
+          workspaceQuery.createOrUpdate(v1Workspace),
+          WorkspaceMigrationMonitor.schedule(v1Workspace)
+        )
+      }
+
+      val migration = runAndWait(
+        WorkspaceMigrationMonitor.workspaceMigrations
+          .filter(_.workspaceId === v1Workspace.workspaceIdAsUUID).result
+      )
+        .head
+
+      val writeAction = WorkspaceMigrationMonitor.deleteWorkspaceBucket(
+        migration,
+        v1Workspace,
+        new MockGoogleStorageService[IO] {
+          override def deleteBucket(googleProject: GoogleProject, bucketName: GcsBucketName, isRecursive: Boolean, bucketSourceOptions: List[Storage.BucketSourceOption], traceId: Option[TraceId], retryConfig: RetryConfig): fs2.Stream[IO, Boolean] =
+            fs2.Stream.emit(true)
+        }
+      ).unsafeRunSync
+
+      runAndWait(writeAction) shouldBe()
+
+      val workspaceBucketDeleted = runAndWait(
+        WorkspaceMigrationMonitor.workspaceMigrations
+          .filter(_.id === migration.id)
+          .map(_.workspaceBucketDeleted)
+          .result
+          .map(_.head)
+      )
+
+      workspaceBucketDeleted shouldBe defined
+    }
+  }
+
+  "deleteTemporaryBucket" should "delete the temporary bucket and record when it was deleted" in {
+    val sourceProject = "general-dev-billing-account"
+    val sourceBucket = "az-leotest"
+    val destProject = "terra-dev-7af423b8"
```

Comment:
I think the above "real" values are artifacts from when were were actually testing this thing against real buckets.  Now that this test is a proper unit test again, can we change these values to ones that are clearly "test" values?  I know it technically works, but I don't want future maintainers to come in here and think this test needs very specific values or projects to be used here.

Reply:
Oh perhaps it shouldnt be...

Reply:
I've removed all the redefinitions - they weren't needed